web/src/index.ejs

```diff
@@ -48,6 +48,8 @@
   <link rel="preload" href="/fonts/raleway/raleway-v12-latin_latin-ext-regular.woff2" as="font" type="font/woff2"
         crossorigin />
 
+  <link rel="stylesheet" href="/fonts/varela-round/varela-round.css" type="text/css" />
```

Comment:
Are fonts only loaded on demand?

Reply:
Yes they are. I was not sure whats the best way to handle it. It would be nice if Integreat would not load Aschaffenburg fonts.

Reply:
I think we should remove the preload for all fonts for now until we have a good solution.